graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/context/DgsContext.kt

```diff
@@ -125,7 +125,7 @@ open class DgsContext(val customContext: Any? = null, val requestData: DgsReques
 
         @JvmStatic
         fun getRequestData(batchLoaderEnvironment: BatchLoaderEnvironment): DgsRequestData? {
```

Comment:
I'm not sure if this change is completely safe/backwards compatible since the API documentation does not specify a type for the context. Perhaps it would be better to check the type first, i.e. whether it is DgsContext or GraphQLContext and return that accordingly?

Reply:
Thanks for the comment. I've done so (and fixed `getCustomContext(batchLoaderEnvironment)` too).